charmcraft.yaml

```diff
@@ -20,3 +20,4 @@ parts:
       - pkg-config
     charm-binary-python-packages:
       - psycopg2-binary==2.9.6  # renovate
```

Comment:
Going forward we should consider porting the other charm libs to psycopg3

Reply:
I thought we should get rid of 3rd party binaries here, either:
 * build from sources each time => slow
 * build and cache psycopg2-binary on Canonical side.

Reply:
Building psycopg from source isn't straightforward, as it needs SO dependencies installed as well (libpq).  I think we should first switch to a single version before trying to figure out how to do it.